filing_history_descriptions.yml

```diff
@@ -119,9 +138,9 @@ description:
     'liquidation-voluntary-creditors-voluntary-winding-up-scotland' : "**Confirmation of creditors' voluntary winding up**"
     'liquidation-voluntary-liquidators-statement-of-receipts-payments-scotland' : "**Statement of receipts and payments**"
     'liquidation-voluntary-appointment-of-liquidator' : "**Appointment of a voluntary liquidator**"
-    'liquidation-voluntary-arrangement-meeting-approving-companies-voluntary-arragement' : "**Notice to Registrar of companies voluntary arrangement** taking effect"
+    'liquidation-voluntary-arrangement-meeting-approving-companies-voluntary-arrangement' : "**Notice to Registrar of companies voluntary arrangement** taking effect"
     'liquidation-voluntary-arrangement-revocation-or-suspension' : "**Order of revocation or suspension of voluntary arrangement**"
-    'liquidation-cva-supervisors-abstracts-of-receipts-payments-with-brought-down-date' : "**Voluntary arrangement supervisor's abstract of receipts and payments** to {brought_down_date}"
+    'liquidation-cva-supervisors-abstract-of-receipts-payments-with-brought-down-date' : "**Voluntary arrangement supervisor's abstract of receipts and payments** to {brought_down_date}"
```

Comment:
And again, there are a few more but you get the gist, even though you are correcting the spelling, developers may have already coded against these enumerations and hence will break their services.